src/Control/ShoppingCartController.php

```diff
@@ -748,29 +750,32 @@ public function placeorderformember(HTTPRequest $request)
     }
 
     /**
-     * This can be used by admins to log in as customers
-     * to place orders on their behalf...
+     * This can be used by admins to log in as customers to place orders on
+     * their behalf...
      *
-     * @param SS_HTTPRequest $request
+     * @param \SilverStripe\Control\HTTPRequest $request
```

Comment:
Should we add use statement at the top?  I am not sure what the best way to do this is.